### ISSUE TITLE
Artifact-rejected RTs not rejected during RT export

### ISSUE THREAD

**Issue by jasonarita** [lucklab/erplab#28]
### Reported by Marianna Eddy

> Marianna Eddy marianna.eddy@tufts.edu  to erplab 
Mar 27

> I'm finding that when I use the export reaction times function in ERPLAB and choose the option "Discard RTs with marker AR flags" I'm still getting RTs for trials that have been marked as rejected. When I create averaged ERPs the artifact rejected trials are not included so I know artifact trials are marked, but they don't seem to be being picked up when exporting RT. Any suggestions on how to fix this would be great.

**Comment by jasonarita** [comment]
Received example datasets:
- `example_subject_epoched.set`
- `example_subject_epoched.fdt`

Files are already event list created, Binned, Epoched, although no artifact detection flags have been set.

- Running artifact rejection `Moving window peak-to-peak` to create `example_subject_epoched_ar.set` with the following artifact-flagged bins:
```
%Equivalent command:
EEG  = pop_artmwppth( EEG , 'Channel',  1:36, 'Flag',  1, 'Threshold',  100, 'Twindow', [ -99.6 697.3], 'Windowsize',  200, 'Windowstep',...
  100 ); % GUI: 28-May-2015 15:01:19
--------------------------------------------------------------------------------------------------------------------------------------------
Go back to ERPLAB menu

Done.  What's next?

Saving dataset...
Creating a new ALLEEG dataset 2
Done.
Bin   #(%) accepted   #(%) rejected    # F2    # F3    # F4    # F5    # F6    # F7    # F8
  1     238( 94.4)       14(  5.6)       0       0       0       0       0       0       0
  2      88( 92.6)        7(  7.4)       0       0       0       0       0       0       0
  3     150( 95.5)        7(  4.5)       0       0       0       0       0       0       0
  4     927( 93.3)       67(  6.7)       0       0       0       0       0       0       0
  5      59( 92.2)        5(  7.8)       0       0       0       0       0       0       0
  6      59( 95.2)        3(  4.8)       0       0       0       0       0       0       0
  7     464( 93.4)       33(  6.6)       0       0       0       0       0       0       0
  8      29( 93.5)        2(  6.5)       0       0       0       0       0       0       0
  9      91( 95.8)        4(  4.2)       0       0       0       0       0       0       0
 10     463( 93.2)       34(  6.8)       0       0       0       0       0       0       0
 11      33( 91.7)        3(  8.3)       0       0       0       0       0       0       0
 12      50( 94.3)        3(  5.7)       0       0       0       0       0       0       0
 13     329( 92.9)       25(  7.1)       0       0       0       0       0       0       0
 14      24( 92.3)        2(  7.7)       0       0       0       0       0       0       0
 15       9(100.0)        0(  0.0)       0       0       0       0       0       0       0
 16     135( 94.4)        8(  5.6)       0       0       0       0       0       0       0
 17      10( 90.9)        1(  9.1)       0       0       0       0       0       0       0
 18      75( 94.9)        4(  5.1)       0       0       0       0       0       0       0
 19     326( 92.1)       28(  7.9)       0       0       0       0       0       0       0
 20      19( 95.0)        1(  5.0)       0       0       0       0       0       0       0
 21      16(100.0)        0(  0.0)       0       0       0       0       0       0       0
 22     137( 95.8)        6(  4.2)       0       0       0       0       0       0       0
 23      81( 96.4)        3(  3.6)       0       0       0       0       0       0       0
 24      28( 90.3)        3(  9.7)       0       0       0       0       0       0       0
 25      53(100.0)        0(  0.0)       0       0       0       0       0       0       0
 26      79( 94.0)        5(  6.0)       0       0       0       0       0       0       0
 27      34( 94.4)        2(  5.6)       0       0       0       0       0       0       0
 28      45( 93.8)        3(  6.2)       0       0       0       0       0       0       0
 29      78( 92.9)        6(  7.1)       0       0       0       0       0       0       0
 30      26( 92.9)        2(  7.1)       0       0       0       0       0       0       0
 31      52( 92.9)        4(  7.1)       0       0       0       0       0       0       0
 32      39( 92.9)        3(  7.1)       0       0       0       0       0       0       0
 33      19( 86.4)        3( 13.6)       0       0       0       0       0       0       0
 34      20(100.0)        0(  0.0)       0       0       0       0       0       0       0
 35      39( 92.9)        3(  7.1)       0       0       0       0       0       0       0
 36      25( 96.2)        1(  3.8)       0       0       0       0       0       0       0
 37      14( 87.5)        2( 12.5)       0       0       0       0       0       0       0
 38      40( 95.2)        2(  4.8)       0       0       0       0       0       0       0
 39      15( 93.8)        1(  6.2)       0       0       0       0       0       0       0
 40      25( 96.2)        1(  3.8)       0       0       0       0       0       0       0
 41      42(100.0)        0(  0.0)       0       0       0       0       0       0       0
 42       9(100.0)        0(  0.0)       0       0       0       0       0       0       0
 43      33(100.0)        0(  0.0)       0       0       0       0       0       0       0
 44      40( 95.2)        2(  4.8)       0       0       0       0       0       0       0
 45       9( 90.0)        1( 10.0)       0       0       0       0       0       0       0
 46      31( 96.9)        1(  3.1)       0       0       0       0       0       0       0
 47      38( 90.5)        4(  9.5)       0       0       0       0       0       0       0
 48      11( 91.7)        1(  8.3)       0       0       0       0       0       0       0
 49      27( 90.0)        3( 10.0)       0       0       0       0       0       0       0
 50      26( 92.9)        2(  7.1)       0       0       0       0       0       0       0
 51      14(100.0)        0(  0.0)       0       0       0       0       0       0       0
 52      21( 87.5)        3( 12.5)       0       0       0       0       0       0       0
 53      17( 94.4)        1(  5.6)       0       0       0       0       0       0       0
 54      18( 94.7)        1(  5.3)       0       0       0       0       0       0       0
 55      21( 91.3)        2(  8.7)       0       0       0       0       0       0       0
 56     106( 90.6)       11(  9.4)       0       0       0       0       0       0       0
 57      11(100.0)        0(  0.0)       0       0       0       0       0       0       0
 58      30( 96.8)        1(  3.2)       0       0       0       0       0       0       0
 59       7( 87.5)        1( 12.5)       0       0       0       0       0       0       0
 60      31( 91.2)        3(  8.8)       0       0       0       0       0       0       0
 61       5(100.0)        0(  0.0)       0       0       0       0       0       0       0
 62      37(100.0)        0(  0.0)       0       0       0       0       0       0       0
 63      15(100.0)        0(  0.0)       0       0       0       0       0       0       0
 64      20(100.0)        0(  0.0)       0       0       0       0       0       0       0
 65     135( 95.7)        6(  4.3)       0       0       0       0       0       0       0
 66      24( 92.3)        2(  7.7)       0       0       0       0       0       0       0
 67       9(100.0)        0(  0.0)       0       0       0       0       0       0       0
 68     135( 94.4)        8(  5.6)       0       0       0       0       0       0       0
 69       7(100.0)        0(  0.0)       0       0       0       0       0       0       0
 70      27( 93.1)        2(  6.9)       0       0       0       0       0       0       0
 71     127( 90.7)       13(  9.3)       0       0       0       0       0       0       0
 72      19( 95.0)        1(  5.0)       0       0       0       0       0       0       0
 73      16(100.0)        0(  0.0)       0       0       0       0       0       0       0
 74     137( 95.8)        6(  4.2)       0       0       0       0       0       0       0
____________________________________________________________________________________________________
Total  5698( 93.8)      376(  6.2)       0       0       0       0       0       0       0

-------------------------------------------------------------------------------------------------------------------------------------------
%Equivalent command:
[EEG, tprej, acce, rej, histoflags ] = pop_summary_AR_eeg_detection(EEG,'none'); % GUI: 28-May-2015 15:05:26
```

**Comment by jasonarita** [comment]
This appears to be the same issue documented and fixed in issue #14. I've tested the fix with Marianna's test files and the number of exported RTs are the same as the number of accepted trials reported in the artifact detection summary, so it looks like it's working. 

Also, I emailed Marianna re: the fix and gave her the updated MARKARTIFACTS.M function.
